.github/workflows/update-files.yml

```
@@ -0,0 +1,112 @@
+name: Update Valkey Extensions
+
+on:
+  repository_dispatch:
+    types: [valkey-release, json-release, bloom-release, search-release]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release'
+        required: true
+      module:
+        description: 'Module to release'
```

```suggestion
        description: 'Valkey Bundle component to be released'
```
does it sound right??

---

I think saying `Module to be released` sounds better. WDYT?